Source/WebCore/platform/graphics/Icon.h

```diff
@@ -61,6 +66,12 @@ class Icon : public RefCounted<Icon> {
     static Ref<Icon> create(HICON hIcon) { return adoptRef(*new Icon(hIcon)); }
 #endif
 
+#if PLATFORM(GTK)
+    WEBCORE_EXPORT static RefPtr<Icon> create(GIcon*);
+
+    GRefPtr<GIcon> icon() const { return m_icon; };
```

Comment:
There's no need to return ownership here:

```suggestion
    GIcon* icon() const { return m_icon.get(); };
```